include/dxc/DXIL/DxilInstructions.h

```diff
@@ -8042,498 +8042,234 @@ struct DxilInst_TextureStoreSample {
   void set_sampleIdx(llvm::Value *val) { Instr->setOperand(10, val); }
 };
 
-/// This instruction Annotate a wave matrix pointer with the type information
-struct DxilInst_WaveMatrix_Annotate {
+/// This instruction Deprecated
+struct DxilInst_Deprecated0 {
```

Comment:
is there no way to do this that doesn't involve having a load of stub instruction structs?